Packages/vcs/Lib/configurator.py

```diff
@@ -730,155 +664,42 @@ def create(self):
-    x, y = point
-
-    attrs = [
-        "file",
-        "function",
-        "logicalmask",
-        "transformation",
-        "source",
-        "dataname",
-        "title",
-        "units",
-        "crdate",
-        "crtime",
-        "comment1",
-        "comment2",
-        "comment3",
-        "comment4",
-        "xname",
-        "yname",
-        "zname",
-        "tname",
-        "xunits",
-        "yunits",
-        "zunits",
-        "tunits",
-        "xvalue",
-        "yvalue",
-        "zvalue",
-        "tvalue",
-        "mean",
-        "min",
-        "max",
-        "xtic1",
-        "xtic2",
-        "xmintic1",
-        "xmintic2",
-        "ytic1",
-        "ytic2",
-        "ymintic1",
-        "ymintic2",
-        "xlabel1",
-        "xlabel2",
-        "ylabel1",
-        "ylabel2",
-        "box1",
-        "box2",
-        "box3",
-        "box4",
-        "line1",
-        "line2",
-        "line3",
-        "line4",
-        "legend",
-        "data",
-    ]
-
-    intersecting = None
-
-    for attr in attrs:
-        attribute = getattr(template, attr)
-        if attribute.priority == 0 or (intersecting is not None and intersecting.priority > attribute.priority):
-            # 0 is turned off
-            continue
-        t_x = safe_get(attribute, "x")
-        t_y = safe_get(attribute, "y")
-
-        if t_x is not None or t_y is not None:
-            if t_x is not None and t_y is not None:
-                # It's probably a text blob
-                if is_label(attribute):
-                    text = editors.label.get_label_text(attribute, dp)
-                    if text == '':
-                        continue
-                    if editors.label.inside_label(attribute, text, x, y, *window_size):
-                        intersecting = attribute
-                elif is_point_in_box((x, y), ((t_x - fudge, t_y - fudge), (t_x + fudge, t_y + fudge))):
-                    intersecting = attribute
-            else:
-                pass
-                """ Uncomment to enable axis editors
-                # It's probably the labels for an axis
-                if t_x is not None and t_x < x + fudge and t_x > x - fudge:
-                    intersecting = attribute
-                elif t_y is not None and t_y < y + fudge and t_y > y - fudge:
-                    intersecting = attribute
-                """
-        else:
-            pass
-            """ Uncomment to reenable the box editors
-            x1 = safe_get(attribute, "x1")
-            y1 = safe_get(attribute, "y1")
-            x2 = safe_get(attribute, "x2")
-            y2 = safe_get(attribute, "y2")
-
-            if None in (x1, y1, x2, y2):
-                continue
-            else:
-                if is_point_in_box((x, y), ((min(x1, x2), min(y1, y2)), (max(x1, x2), max(y1, y2)))):
-                    intersecting = attribute
-            """
-    return intersecting
+def is_label(key):
```

Comment:
While this, perhaps, is not the ultimate way for determining if an actor is a label in VCS, it's quite accurate as things currently stand.